rocky/rocky/templates/partials/cat_paws.html

```diff
@@ -0,0 +1,6 @@
+<ul class="level-indicator l{% if level < 0 %}0{% else %}{{ level }}{% endif %} {{ custom_class }}">
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+</ul>
```

Comment:
We already had "scan_level_indicator.html" which does this, but more dynamically (instead of having the `li`'s static, it loops over the amount of `scan_levels`. I think you can use that partial and remove this one.

Reply:
The scan level indicator is used differently :
<img width="516" alt="image" src="https://user-images.githubusercontent.com/30832687/225025076-480bf11a-9643-4f4d-ba70-9b84b9e1fd0a.png">

Reply:
They are both ul's with li's that represent cat pawns? Don't really see what's different...

Reply:
`scan_level_indicator` is looping without radio button in the loop

Reply:
Why not do it something like this?

`<input type="radio"
             name="{{ field.name }}"
             value="{{ choice.data.value }}"
             id="id_{{ field.name }}_{{ forloop.counter0 }}"
             {% if choice.data.attrs.disabled %} disabled{% endif %}
             {% if choice.data.attrs.checked %} checked{% endif %}/>
      <label>
          {% include 'partials/scan_level_indicator.html' with value=forloop.counter0 %}
      </label>
      `
      
That way you get pawns rendered as an UL with LI's with unfilled pawns where needed. That's exactly what this component was meant to do.